client/state/data-layer/wpcom/concierge/schedules/appointments/reschedule/to-api.js

```diff
@@ -0,0 +1,3 @@
+export default ( { beginTimestamp } ) => ( {
+	begin_timestamp: beginTimestamp / 1000, // convert to UNIX timestamp.
```

Comment:
Two things 😄:
* I'm sure this was just copy/pasted but this isn't actually a valid unix timestamp since the division will yield a float
* This logic has been repeated a few times. Would it make since to just do the truncation of the last 3 digits a single time before sending it through redux?

Reply:
> I'm sure this was just copy/pasted but this isn't actually a valid unix timestamp since the division will yield a float

Nice catch but it seems that the numbers are rounded before being sent to the server. However to be extra-safe I added a `Math.ceil`.

> This logic has been repeated a few times. Would it make since to just do the truncation of the last 3 digits a single time before sending it through redux?

The logic is used only in the transformers that prepare the request' body to be sent to the server because the server uses UNIX timestamp. I would rather keep them javascript date compliant in redux and just convert when talking to the server.